NugetLightConstant.cs

```diff
@@ -28,7 +28,7 @@ public static class NuGetConstant
         /// <summary>
         /// Version of this provider
         /// </summary>
-        public static readonly string ProviderVersion = "2.8.5.210";
+        public static readonly string ProviderVersion = "2.8.5.211";
```

Comment:
can you make major change 3.0.0.1 for example?

Reply:
dfb31a1